docs/version_history.rst

```diff
@@ -0,0 +1,12 @@
+================
```

Comment:
I think this file must to be removed

Reply:
you are correct! Forgot to jeje. Doing it now.